recipes/opentracing_instrumentation/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "opentracing_instrumentation" %}
+{% set version = "2.2.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "d7d5cfa5862048f824e7cc33289c359c7d972287d45806701aeda455652513ee" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_value }}
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - futures  # [py2k]
```

Comment:
Seems to not be constrained in the package itself. Sure it needs a selector here? Also if we drop the selector, this can be `noarch`. :)

Reply:
futures is a py2 package only. 
The GH repo for opentracing_instrumentation marks the py3 support as alpha. Should I restrict this package to py27 only then?

Reply:
Hmm... actually, it's not a real dependency at v2.2.0: https://github.com/uber-common/opentracing-python-instrumentation/blob/2.2.0/setup.py
Maybe I should make this py27 only then?

Reply:
Doesn't look like they support Python 3 in that release (based on the trove classifiers). 😒 So it sounds like it is Python 2 only for this release at least.

Reply:
I've updated the recipe to make it py2 only.